src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/SearchBox/BitSearchBoxDemo.razor

```diff
@@ -43,4 +43,14 @@
             <BitSearchBox Placeholder="Search" IsEnabled="false"></BitSearchBox>
         </ExamplePreview>
     </ComponentExampleBox>
+
+    <ComponentExampleBox Title="SearchBox Validation" HTMLSourceCode="@example6HTMLCode" CSharpSourceCode="@example6CSharpCode" ExampleId="example6">
+        <ExamplePreview>
+            <EditForm Model="validationSearchBoxModel">
```

Comment:
where is the description of this sample?
@maryamhdr plz help to add the description 😊